locale/en/knowledge/javascript-conventions/what-is-the-arguments-object.md

```diff
@@ -24,41 +24,63 @@ property of `arguments` is callee. This always refers to the function
 currently being executed. Here's an example that illustrates the
 properties of `arguments`.
 
-    var myfunc = function(one) {
-      arguments.callee === myfunc;
-      arguments[0] === one;
-      arguments[1] === 2;
-      arguments.length === 3;
-    }
+```js
+const myfunc = function(one) {
+  arguments.callee === myfunc;
```

Comment:
`arguments.callee` has been deprecated in ES5 strict mode:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/arguments/callee